File: Day1/Ruby/day1.rb
depth = File.open("sonar.txt", chomp: true).map(&:to_i)
def depth_counter(depth)
    count=0
      (0...depth.length-1).each do |i|
        prev = depth[i]
        nxt = depth[i+1]
        if nxt > prev
            count +=1
        end
    end
    count
end

p depth_counter(depth)

File: Python/Day1/day1.py
with open("sonar.txt") as f:
    lines=f.read()
    depth=lines.split("\n")
def depth_counter(depth):
    count=0
    for i in range(len(depth)-1):
        prev = int(depth[i])
        next = int(depth[i+1])
        if next > prev:
            count +=1
    return(count)

print(depth_counter(depth))
